nvflare/apis/fl_constant.py

```diff
@@ -307,7 +314,10 @@ class WorkspaceConstants:
 
     SERVER_STARTUP_CONFIG = "fed_server.json"
     CLIENT_STARTUP_CONFIG = "fed_client.json"
-    JOB_META_FILE = "meta.json"
+    JOB_META_FILE = JobConstants.META_FILE
+
+    SERVER_APP_CONFIG = JobConstants.SERVER_JOB_CONFIG
+    CLIENT_APP_CONFIG = JobConstants.CLIENT_JOB_CONFIG
```

Comment:
Do we really need to duplicate these constants?
Or we can just import JobConstants if we need to use these stuff?